### PR TITLE
Changed how the ° and unit system are displayed.

### PR DESCRIPTION
**Pull request by cruncharoo**
Subsequent formatting to make the above change look cleaner.

### CODE REVIEW

**Comment by cruncharoo** [comment]
I didn't make any changes to package-lock.json; I'm not sure how that got included in my PR.

File: package-lock.json
Comment (drillskibo) [inline package-lock.json]: Please remove this as CJ does not want *any* change to the package-lock.json that is not from themselves, to prevent any kind of hack attempt.

**Review by drillskibo — CHANGES_REQUESTED** [review]
Could you please provide a live test url, or at least a screenshot, and explain your changes for clarity ?

**Comment by cruncharoo** [comment]
> Could you please provide a live test url, or at least a screenshot, and explain your changes for clarity ?

Here you go. I changed the sizing and put both degree symbol and unit in superscript. This is similar to how both are shown on various other sites and I think looks cleaner.

image

File: package-lock.json
Comment (cruncharoo) [reply package-lock.json]: Thanks I will try to do that now. Still very new to github. I didn't edit that file either so I am confused on how it got included in this.

Comment (drillskibo) [reply package-lock.json]: Just remove the line from your lock, then add, commit and push to your repo, it will automatically add the commit to this pr 😄

**Review by drillskibo — APPROVED** [review]
LGTM ! Still needs manual review from CJ locally before merging
(Take care of the lock issue if you can, if not it's not a big deal but would be preferred)

File: package-lock.json
Comment (cruncharoo) [reply package-lock.json]: Thank you. Figured it out!

**Pull request closed without merging**
